index.html

```diff
@@ -14,15 +14,49 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400;600&display=swap"
+    rel="stylesheet">
 </head>
 <body>
   <header>
     <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1 class="white-text">Code Journal</h1>
+        </div>
+      </div>
     </div>
   </header>
+
   <main class="container">
-    <div data-view="entry-form"></div>
+
+    <div data-view="entry-form">
+      <h2>New Entry</h2>
+      <form action="" class="row">
+        <div class="column-full column-half journal-photo"><img src="images/placeholder-image-square.jpg" alt="place holder for image"></div>
```

Comment:
You don't need to apply `column-full` here, because there's already CSS in `layout.css` that will make this `width: 100%` on mobile & tablet